massgov/openmass
# Fix missing fields in AcquiaSyslogHandler. Stop custom X-Content-Type-Options header handling

- Fixes Acquia watchdog logging since Monolog upgrade
- Brings back briefer timestamp in local dev log messages
- Removes custom handling of X-Content-Type-Options header since core already sets that ([in duplicate, but its harmless](https://www.drupal.org/project/drupal/issues/2854817)).

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-19670


**To Test:**
- Go to [Log Streaming page for Dev env](https://cloud.acquia.com/a/develop/applications/ff8ed1de-b8bc-48a4-b316-cd91bfa192c4/environments/26645-ff8ed1de-b8bc-48a4-b316-cd91bfa192c4/logs/stream) where this branch is deployed. Enable the `drupal-watchdog` stream by clicking the `Streams` link.
- SSH into dev environment and run: `drush php:eval "\Drupal::logger('bar')->log(3, 'foo');"`
- On Log Stream page, observe that the base_url and timestamp fields are populated. For example, see `http://default|1597260558` segments below:

```
drupal-watchdog staging-21435 Aug 12 19:29:18 staging-21435 massgovdev[17842]: http://default|1597260558|bar|ERROR|127.0.0.1|http://default/||0||foo
```
- If you want, do above experiment on stage and you will see that those fields are not populated.



**Screenshots/GIFs:**
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
